R/shift.R

```diff
@@ -24,16 +24,16 @@ shift = function(x, n=1L, fill=NA, type=c("lag", "lead", "shift"), give.names=FA
   ans
 }
 
-nafill = function(x, type=c("const","locf","nocb"), fill=NA, verbose=getOption("datatable.verbose")) {
+nafill = function(x, type=c("const","locf","nocb"), fill=NA, nan_is_na=TRUE, verbose=getOption("datatable.verbose")) {
```

Comment:
argument name is too long IMO
Maybe we could align to API in https://github.com/Rdatatable/data.table/compare/fast-isna function?

Reply:
I like it, will also add a `util` function to handle such arg.

Reply:
I think it'll be easier to use a `bool` internally. How about `nan_is_na` for internal use & `nan` for the function API only?

Reply:
Check latest commit for sample approach